### DISCUSSION TITLE
Upgrade notes for PimCore 6.9

### DISCUSSION THREAD

**Discussion by tapli** [pimcore/pimcore#14628] [Q&A / Support] [answered]
I am not sure if this is the right forum to ask this question or not. I am very new to PimCore and learning it. There is a question in the release notes 6.9 and it is very confusing to me. I went through 6.9 release notes in the documentation. And yet I am not able to figure out the correct options.

Here is the question with options 

What are relevant upgrade notes for Pimcore 6.9?

1.  Additional templating helper were introduced.
2.  Normalize/Denormalize will replace marshal/unmarshal of data object field definitions.
3.  Many to one relations are now allowed in reverse relations.
4.  Elasticsearch 7 support was introduced.
5.  New settings store for simple key value data was introduced.
6.  Geopoint data types were removed from data objects.

Here is my analysis. 

1 - Per release note, there are some additional helpers introduced. No reference found on this topic. Does it mean **(https://github.com/pimcore/pimcore/pull/7415)** ?

2 - This is also listed in the release note - **(https://github.com/pimcore/pimcore/issues/8488)**

3 - This is correct - **https://github.com/pimcore/pimcore/pull/8037**

4 - Per course slide and per release notes, this is correct ( **https://github.com/pimcore/pimcore/issues/8096**)

5- Per course note this is also correct -  **(https://github.com/pimcore/pimcore/pull/8332)**

6 - This is correct - **https://github.com/pimcore/pimcore/pull/8306**

Based on the above analysis, I think the answer should be 2, 3, 4, 5 and 6. But that is not correct. If I include # 1 then also its not correct. 
Is there an issue with this question answers set? If not, then can someone please guide me? I would appreciate it.
Thank you.

**Comment by tapli** [comment, marked as answer]
@fashxp Can you please guide me? I have gone through the course material many times and yet not getting the answers correctly. I feel something is not right about the answer sets.

**Reply by tapli** [reply]
@fashxp I have updated the response with the actual git hub issue numbers.

**Reply by dvesh3** [reply]
I think your answer is close enough. 6 is also wrong as Geopoint data type was not removed from dataobjects. Only it started using `Data\GeoCoordinates` for storing longitude & latitude values instead of `Data\Geopoint`.

**Reply by tapli** [reply]
Hi @dvesh3.

Per course material, it is Deprecate. See the attached image.

![image](https://user-images.githubusercontent.com/21690536/226379788-d5bb934c-b9c5-41b2-a7a9-094e962afa4e.png)

**Reply by dvesh3** [reply]
Just for clarity, there's a difference between `Pimcore\Model\DataObject\ClassDefinition\Data\Geopoint` and `Pimcore\Model\DataObject\Data\Geopoint`, the former is the datatype class and the later is data class used for persisting data for data types.  The data class was class **deprecated** in 6.9 and **removed** in version 10.

The answer is wrong as it says the data type (`Pimcore\Model\DataObject\ClassDefinition\Data\Geopoint`) is removed.

**Reply by tapli** [reply]
@dvesh3 Thanks for clarifying the answer. With this I tried 1, 2, 3, 4 and 5 as correct answers. And it said invalid answers. If my understanding of answer 1 is incorrect then I removed that and answered 2,3,4 and 5. Yet it says invalid answers.

**Reply by tapli** [reply]
@dvesh3 the correct answers are 1,2,3,4 and 5. Thanks.
